src/Service.Host/client/src/components/__tests__/DateUpdater.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import { Grid, Row, Col, Button } from 'react-bootstrap';
+import InlineDatePicker from '../common/InlineDatePicker';
+import { shallow } from 'enzyme'
+import { DateUpdater } from '../DateUpdater';
+import moment from 'moment';
+
+
+
+describe('<DateUpdater />', () => 
+{
+    const before = moment("1995-03-28");
+    const after = moment("2018-03-28");
+
+    let onChange = jest.fn();
+
+    const defaultProps = {
+        displayOnly: false,
+        value: before,
+        onChange: onChange
+    }
+
+
+    
+    
+    it('renders an update Button and Picker if not displayOnly', () =>
```

Comment:
You can nest describes to make separate 'stories' through the test. 
So you could have had one describe for the display only and one for normal. 
Then you can just define the wrapper once in a beforeEach method and write a series on tests on it.